integration/fuse/src/test/java/alluxio/fuse/AlluxioFuseFileSystemTest.java

```diff
@@ -84,19 +106,6 @@ public void create() throws Exception {
   }
 
   @Test
-  public void createWrongFlags() throws Exception {
```

Comment:
If these flags are supported now, could we change the test to validate that the flags are supported?

Reply:
The flags actually are not so related. For example, in mac the flags are set to read-write, even though the shell command that opens the file only use read-only or write-only.
So the purpose is to remove this false preventive check, but fail the operation late, only when it violates the Alluxio's semantics, which will trigger Alluxio exceptions

Reply:
ack